.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,21 +2,28 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: category:BUG
+labels: ''
 assignees: ''
 
 ---
 
 **Describe the bug**
+<!-- This form is used to report software defects such as inaccurate/erratic behavior, crashes, etc. If you have general questions about using the software or are encountering difficulties with installation, please create a topic in the forum (see the "Discussions" tab at the top of this page). -->
```

Comment:
I'm worried the nudge to go to the Discussions page might be too subtle. Especially because, if someone is thinking of creating an issue for a non-bug question, they'll arrive here first:

![Screenshot from 2020-12-19 09-56-16](https://user-images.githubusercontent.com/16181459/102692340-ae703100-41e0-11eb-9d0b-ba9e5cd4db2b.png)

They might just choose "Open a blank issue" and never even see this little note here about Discussions.

What do you think about creating a third (dummy) issue template called "Everything else" or "Questions and concerns" or something like that, then in the description, include a link to the Discussions page? Basically to say "don't open a blank issue, go here instead", to fill the gaps in GitHub's UI design. :wink: